package.json

```diff
@@ -37,7 +37,7 @@
     "@types/yargs": "^12.0.1",
     "colors": "^1.3.2",
     "fs": "0.0.1-security",
-    "gulp": "^4.0.0",
+    "gulp": "^3.9.1",
```

Comment:
Should this be in this PR?

Reply:
I could see the argument for moving it to its own PR, but I think it's fine to do it here. It's a small enough change that when we want to revert it and move back to 4.0.0, we can just create a new PR to do that rather than trying to revert this PR.